### DISCUSSION TITLE
MooseFS not replicating undegoal chunks? SOLVED

### DISCUSSION THREAD

**Discussion by PC-Admin** [moosefs/moosefs#435] [Q&A]
Strange error recently, it started after getting an invalid chunk in my MooseFS. After deleting the file associated with that I stopped getting errors about the invalid chunk, but now the system isn't replicating undergoal chunks. It does however replicate endangered chunks. Here is a screenshot:

![MooseFS_not_rebalancing](https://user-images.githubusercontent.com/29645145/118759078-e4eecc80-b8a2-11eb-9ef7-66bb7094a444.png)

The master and chunkservers are all Debian 10 running '3.0.115-1~bpo10+1 amd64' of MooseFS. Checked the logs of master and they seem normal. Does anyone know why this might be happening?

SOLVED: After the very imbalanced disks rebalanced more the undergoal chunks started replicating again. It appears that's the default behavior of MooseFS.

**Comment by Blackpaw** [comment]
How many chunkservers do you have? are the disks evenly distributed? could some of them be overfull?

**Reply by PC-Admin** [reply]
3, each with about 60TB of disks, I recently swapped a bunch of disks in and they are currently re-balancing across them, perhaps MooseFS prioritizes that over replicating undergoal chunks?

**Reply by chogata** [reply]
Do you have high speed rebalance on, that is, option HDD_HIGH_SPEED_REBALANCE_LIMIT is uncommented and set to a number > 0 on some or all of your chunkservers? If yes, then only endangered chunks will be replicating until it's off again.

**Reply by PC-Admin** [reply]
No, there are no custom settings on any of these chunkservers.
